airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -142,8 +155,15 @@ def _fetch_dag_runs(
     )
     # Count items
     total_entries = query.count()
+    # sort
+    if sort == 'asc':
+        query = query.order_by(asc(order_by))
```

Comment:
This would break if `sort` was passed by `order_by` wasn't (as order_by has a default of None.)